client/src/components/ReviewEntry.jsx

```diff
@@ -86,6 +86,20 @@ const UserBody = styled.div`
 `;
 UserBody.displayName = 'UserBody';
 
+const VIP = styled.div`
+    text-transform: uppercase;
```

Comment:
is this necessary? since your text is in caps also

Reply:
You're right! I'll take it out